Standards/scs-XXXX-vN-Availability-Zones-Standard.md

```diff
@@ -0,0 +1,204 @@
+But how CSPs apply Availability Zones to the IaaS Layer in one deplyoment may differ widely.
+Therefore this standard will address the minimal requirements that need to be met, when creating Avaiability Zones.
+
+## Terminology
+
+| Term               | Explanation                                                                                                                              |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| Availability Zone  | (also: AZ) internal representation of physical grouping of service hosts, which also lead to internal grouping of resources.             |
+| Fire Zone          | A physical separation in a data center that will contain fire within it. Effectively stopping spreading of fire.                         |
+| PDU                | Power Distribution Unit, used to distribute the power to all physical machines of a single server rack.                                  |
+| Compute            | A generic name for the IaaS service, that manages virtual machines (e.g. Nova in OpenStack).                                             |
+| Network            | A generic name for the IaaS service, that manages network resources (e.g. Neutron in OpenStack).                                         |
+| Storage            | A generic name for the IaaS service, that manages the storage backends and virtual devices (e.g. Cinder in OpenStack).                   |
+| BSI                | German Federal Office for Information Security (Bundesamt für Sicherheit in der Informationstechnik)                                     |
+| CSP                | Cloud Service Provider, provider managing the OpenStack infrastructure.                                                                  |
+
+## Motivation
+
+Redundancy is a non-trivial but relevant issue for a cloud deployment.
+First and foremost it is necessary to increase failure safety through redundancy on the physical layer.
+The IaaS layer as the first abstraction layer from the hardware has an important role in this topic, too.
+The grouping of redundant physical resources into Availability Zones on the IaaS level, gives customers the option to distribute their workload to different AZs which will result in a better failure safety.
+While CSPs already have some similarities in their grouping of physical resources to AZs, there are also differences.
+This standard aims to reduce those differences and will clarify, what customers can expect from Availability Zones in IaaS.
+
+Availability Zones in IaaS can be set up for Compute, Network and Storage separately while all may be referring to the same physical separation in a deployment.
+This standard elaborates the necessity of having Availability Zones for each of these classes.
```

Comment:
I guess "these classes" refers to the "Compute, Network and Storage" in the line above, right?
Would "these resources" better in this context?